### DISCUSSION TITLE
Easly add a custom domain/subdomain and assign a directory

### DISCUSSION THREAD

**Discussion by Kaizodo** [leokhoa/laragon#585] [Ideas]
It would really awesome if we can easly create domain and subdomain and select a directory for it instead of tweaking various config files and hosts file , because not all developers put there php projects in same directory , i have projects in different different directories some or them are on external drive as well, so it would be really awesome where i can simply create a subdomain on laragon app and assing a directory to it

**Comment by kstingel** [comment]
@Kaizodo 
`{tld}` is whatever domain suffix you are using for pretty URLs, `adomain` & `bdomain` are your top level domains

| HDD Path | Site |
|--------|--------|
| `{Laragon Root}/www` | localhost |
| `{Laragon Root}/www/adomain` | adomain.{tld} |
| `{Laragon Root}/www/sub1.adomain` | sub1.adomain.{tld} |
| `{Laragon Root}/www/app1.adomain` | app1.adomain.{tld} |
| `{Laragon Root}/www/bdomain` | bdomain.{tld} |
| `{Laragon Root}/www/blog1.bdomain` | blog1.bdomain.{tld} |

Laragon does not care where your working files are, it only cares about where your hosted files are

> i have projects in different different directories some or them are on external drive as well

I build my webs on my local PC (in Australia), but my `'live'` host is on a machine in America - 
you can't see my latest work until I publish it to my web via FTP

Laragon is not designed to be a poduction server, it's designed to be a staging server 
allowing you to view and test your apps/sites while they are in active development

unlike most staging server solutions, Laragon provides us a way (via `ngrok`), to share our 
work in progress with the wider world and gain feedback prior to live publishing
